### ISSUE TITLE
Filename autocomplete

### ISSUE THREAD

**Issue by LeDieuBisounours** [VSCodeVim/Vim#8096]
Hello,

I would like to be able to use the vim autocomplete Ctrl+x Ctrl+f which is able to autocomplete paths.
It is an important feature to me when making scripts.

Example: for a path (dir1/dir2/dir3/myfile.extension)
When writing path my_function_opens("./dir1/dir2/")
Ctrl+x Ctrl+f shows the content of dir2.

I have seen your codebase and am having trouble trying to run tests and/or docker (am not a ts dev but more of a C++ one).
I am ok with developping the feature and creating a PR afterwards.

Can you just help me get a docker container up and running up to the point where I can change a test of the file lineCompletionProvider.ts to make it pass or fail ?

Basically, I try to docker build and docker run but I get an error of type "ENOENT: npm /app/package.json not found, no such file or directory".
I don't really know much about making a docker container, after that I have some experience with many programming languages and JS so should be ok.

Thanks for your consideration,
Hope I can contribute.
